rosidl_generator_rs/resource/build.rs.em

```diff
@@ -0,0 +1,10 @@
+use std::path::Path;
+
+fn main() {
+	let lib_dir = Path::new("../../../lib")
```

Comment:
Is this expecting a specific directory structure? What would happen with a merge install? i.e. everything installed into the install directory, no separate packages.

Reply:
Yep, the relative path is still the same. I tested it.